views/admin-education-plugin-coupons.php

```diff
@@ -39,7 +39,7 @@
 
 		<section class="simpay-landing-zone__screenshot">
 			<div class="simpay-landing-zone__screenshot-img">
-				<img src="<?php echo esc_url( SIMPLE_PAY_INC_URL . 'core/assets/images/settings/coupons-thumb.png' ); // @phpstan-ignore-line ?>" alt="<?php echo esc_attr__( 'Add new coupon', 'stripe' ); ?>" />
```

Comment:
@prappo also noticed these still don't get picked up in the Grunt text domain switch either.

Reply:
@spencerfinnell I discovered that the `checkdomain` task fails to recognize codes with forward slashes `//` preceding them. However, using a different comment format like `/**` allows the text domain to be replaced. Changing `// @phpstan-ignore-line` to `/** @phpstan-ignore-line */` resolves the issue.

Reply:
Nice find! I haven't seen that be an issue before.